oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthorisationHandler.java

```diff
@@ -317,6 +318,23 @@ public APIGatewayProxyResponseEvent authoriseRequestHandler(
             }
         }
 
+        if (!client.getIsActive()) {
+            String errorMsg =
+                    String.format("Client not active for ClientId: %s", client.getClientID());
```

Comment:
I don't think that `clientId` needs to be explicitly logged as it's included in all log messages by default